modules/custom/openy_system/src/Controller/TermsOfUseController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Drupal\openy_system\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Symfony\Component\HttpFoundation\Response;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+
+/**
+ * Provides output terms.txt output.
+ */
+class TermsOfUseController extends ControllerBase implements ContainerInjectionInterface {
+
+  /**
+   * Provides the terms.txt file.
+   *
+   * @return \Symfony\Component\HttpFoundation\Response
+   *   The terms.txt file as a response object with 'text/plain' content type.
+   */
+  public function content() {
+    $config = \Drupal::config('openy.terms_and_conditions.schema');
```

Comment:
you can use already initialized and available config service in the controller. 
```suggestion
    $config = $this->config('openy.terms_and_conditions.schema');
```